### DISCUSSION TITLE
Dataviz: what data do we want visualised in the white papers?

### DISCUSSION THREAD

**Discussion by RaoOfPhysics** [alan-turing-institute/asg-research-communications#75] [Ideas]
### Summary Sentence

We should prepare a list of what data visualisation we need for the white papers. We will gather a list of potential items to be shown here.

### What kinds of data are we talking about?

- Impact numbers from each project (number of partners, users, deployments, documentation pages produced, pieces of software libraries etc.)
- Data used in / generated from projects themselves
- ??

**N.B.**: These are different from (a) illustrations of ideas/concepts and (b) infographics

### What needs to be done?

- [ ] Collect a list of ideas for data visualisations to accompany the white papers
- [ ] Select the items that will feature in the final white papers

### Who can help?

- Everyone involved with the projects referenced in each white paper
- RAMs
- Later, possibly REG (to explore the _kinds_ of visualisation are best suited to the available data)

**Comment by ZaynabIsmail** [comment]
@RaoOfPhysics Lets start planning this now, looks like a lot will be involved.  Have a think for next week and we can plan for this activity.

**Reply by RaoOfPhysics** [reply]
Let’s! :)

**Comment by dingaaling** [comment]
Other data viz ideas:
- Visualising collaboration: network viz of cross-Turing (or cross-UK map of) research collaboration
- Visualising stakeholder diversity: Pie chart of kinds of stakeholders (e.g. gov, industry, academia) that could be broken down further (e.g. local gov, national gov, national body)

**Reply by RaoOfPhysics** [reply]
Nice! I like these. :D

**Comment by malvikasharan** [comment]
Interactive data viz pages (executable code) - when possible. For instance executable notebooks (binderised, Turing data story versions).

**Reply by RaoOfPhysics** [reply]
Absolutely! 👍

**Reply by scotthosking** [reply]
we could take some from our scivision gallary? https://github.com/scivision-gallery

**Reply by RaoOfPhysics** [reply]
Yes, that’s a good idea, Scott. Thanks!

**Reply by tom-andersson** [reply]
We could possibly provide an interactive plot of IceNet's forecasts if interested, see for example: https://the-environmental-ds-book.netlify.app/gallery/modelling/polar-modelling-icenet/polar-modelling-icenet.html#ice-edge

**Reply by RaoOfPhysics** [reply]
Sounds like the kind of thing we’d like, @tom-andersson!
